client/packages.go

```diff
@@ -28,6 +28,7 @@ import (
 
 // Snap holds the data for a snap as obtained from snapd.
 type Snap struct {
+	ID            string    `json:"snap-id"`
```

Comment:
s/snap-id/id/ as well, I think?

We don't have snap-name, snap-status, etc below. It's tempting to call this snap-id because we've been talking about so long as this being a tangible thing we wanted, but it feels like this is still just the Snap.ID, or the {id: ...} in the JSON case.

You asked a question the other day and it's not entirely clear to me I understood the consequences of my answer. If having SideInfo.SnapID is any harder thatn having SideInfo.ID, I wouldn't be against having SideInfo.ID either. We already have SideInfo.OfficialName which is in fact just "name" in JSON.

Perhaps let's have a quick chat about this on Monday.